src/invisible/user-guide/invisible-companion-app.md

```diff
@@ -81,7 +81,7 @@ Pupil Monitor will warn you if a device is not in sync.
 More info on the technical implementation and quality of NTP synchronisation can be found in the [Pupil Invisible developer docs](/developer/invisible/#time-synchronization "Pupil Invisible developer docs - time synchronization").
 
 ## Recording transfer
-### Pupil Cloud
+###Pupil Cloud
```

Comment:
Is the removal of the white space on purpose?

Reply:
Not removed on purpose -- I was just trying different configurations of headings.